project.yml

```diff
@@ -11,4 +11,3 @@ gemspec:
     - [rspec, "~> 3.7"]
     - [simplecov, "~> 0.17.1"]
   runtime_dependencies:
```

Comment:
I think it's better to remove this key or it may break

Reply:
@flash-gordon Removed. How to reflect these changes into `dry-cli.gemspec`?

Reply:
@jodosha it'll be updated automatically after merging